_extensions/partials/quarto-partials.lua

```diff
@@ -58,9 +58,22 @@ function copy(obj, seen)
   return res
 end
 
+local function normalize_quarto_path(file)
+  local prefix = ""
+  local path = pandoc.utils.stringify(file)
+  if string.sub(path, 1, 1) == "/" then
+    prefix = os.getenv("QUARTO_PROJECT_ROOT")
+  end
```

Comment:
Should we carve out an exception for people who want system-absolute paths? Or is that just generally a bad idea? (If we did, it'd be for paths that start with `//`.)

Reply:
I'm on the fence about the badness of the idea, but it's not too hard to implement both a `//` and `file:///` catch.

Reply:
addressed in [609a816](https://github.com/gadenbuie/quarto-partials/pull/7/commits/609a8163ffd1cab0b5d5bd1ba18af1103afe5522)